tracing-subscriber/src/fmt/time/time_crate.rs

```diff
@@ -49,6 +49,23 @@ pub struct UtcTime<F> {
 
 #[cfg(feature = "local-time")]
 impl LocalTime<well_known::Rfc3339> {
+    /// Returns a formatter that formats the current [local time] in the
+    /// [RFC 3339] format (a subset of the [ISO 8601] timestamp format),
+    /// but with a variable number of decimal digits.
+    ///
+    /// Use [`LocalTime::iso_8601`] instead.
+    ///
+    /// [local time]: https://docs.rs/time/0.3/time/struct.OffsetDateTime.html#method.now_local
+    /// [RFC 3339]: https://datatracker.ietf.org/doc/html/rfc3339
+    /// [ISO 8601]: https://en.wikipedia.org/wiki/ISO_8601
+    #[deprecated(note = "this has a variable number of decimal digits, use `LocalTime::iso_8601()` instead")]
```

Comment:
I don't know if it's necessary to deprecate this API. It seems possible some users will actually want to use the RFC 3339 format rather than ISO 8601? Should we just ensure that the documentation clearly indicates the difference between the two formatters, rather than deprecating this one?

Reply:
The RFC 3339 format is covered by the ISO 8601 formatter. RFC 3339 is a subset of ISO 8601. Users who want RFC 3339 can use the ISO 8601 formatter.